db/migrate/20180126150047_create_vote_system.rb

```diff
@@ -0,0 +1,9 @@
+class CreateVoteSystem < ActiveRecord::Migration[5.1]
+  def change
+    create_table :votes do |t|
+      t.belongs_to :user, null: false
+      t.belongs_to :review, null: false
+      t.boolean :vote
```

Comment:
While not required, I would strongly suggesting avoiding a boolean for an attribute that can have three values (upvoted, downvoted, or not voted). `nil` as a value can lead to errors and bugs.